server/testutil/testbazel/testbazel.go

```diff
@@ -52,13 +55,6 @@ func shutdown(ctx context.Context, t *testing.T, workspaceDir string) {
 
 func MakeTempWorkspace(t *testing.T, contents map[string]string) string {
 	workspaceDir := testfs.MakeTempDir(t)
-	t.Cleanup(func() {
```

Comment:
What's the reason to remove this?

Reply:
I found that it slows down the test unnecessarily. It's not really needed locally since bazel defaults to `max_idle_secs=15` when it's run under a bazel test (it checks whether `TEST_TMPDIR` is defined). It's also not needed remotely since we kill the whole container after each test action, which will kill bazel.